selenium/bin/suite_template

```diff
@@ -536,6 +542,15 @@ determine_required_components_excluding_rabbitmq() {
     fi
   }
 }
+initOnly() {
+  for (( i=1; i<=$#; i++)) {
+    if [[ $i != "rabbitmq" ]]; then
+      eval val='$'$i
+      init="init_$val"
+      $init
+    fi
```

Comment:
I'm not sure the `if` condition ever evaluates to false. `$i` expands to numbers e.g. 1, 2, 3, therefore, the comparison is always `<some-number> != "rabbitmq"`, which is always true. If I understand the intention correctly, you can rewrite this loop as:

```bash
for j in "$@"; do
  if [[ "$j" != "rabbitmq" ]]; then
    $j
  fi
done
```

Reply:
You are dead right ! well spotted . Although no one will ever include rabbitmq in the `initOnly` function but it was not correct.